build.gradle

```diff
@@ -1,8 +1,270 @@
+        exclude group: 'jakarta.activation', module: 'jakarta.activation-api'
+    }
+    compile 'com.github.wnameless:json-flattener:0.5.0'
+    compile('com.flipkart.zjsonpatch:zjsonpatch:0.4.4') {
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
+    }
+    compile('org.apache.kafka:kafka-clients:2.5.0') {
+        exclude group: 'org.slf4j', module: 'slf4j-api'
+    }
+    compile('com.onelogin:java-saml:2.5.0') {
+        exclude group: 'org.codehaus.woodstox', module: 'woodstox-core-asl'
+        exclude group: 'joda-time', module: 'joda-time'
+        exclude group: 'org.apache.httpcomponents', module: 'httpclient'
+    }
+    compile('org.opensaml:opensaml-saml-impl:3.4.5') {
+        exclude group: 'joda-time', module: 'joda-time'
+        exclude group: 'commons-collections', module: 'commons-collections'
+    }
+    compile 'commons-collections:commons-collections:3.2.2'
+    compile 'com.jayway.jsonpath:json-path:2.4.0'
+    compile('org.apache.httpcomponents:httpclient:4.5.3') {
+        exclude group: 'org.apache.httpcomponents', module: 'httpcore'
+    }
+    compile('io.jsonwebtoken:jjwt-jackson:0.10.5') {
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
+    }
+    compile 'org.apache.geronimo.specs:geronimo-jta_1.1_spec:1.1.1'
+    compile 'jakarta.annotation:jakarta.annotation-api:1.3.5'
+    compile 'jakarta.jws:jakarta.jws-api:2.1.0'
+    compile('jakarta.xml.soap:jakarta.xml.soap-api:1.4.2') {
+        exclude group: 'jakarta.activation', module: 'jakarta.activation-api'
+    }
+    compile('jakarta.xml.ws:jakarta.xml.ws-api:2.3.3') {
+        exclude group: 'jakarta.activation', module: 'jakarta.activation-api'
+    }
+    compile 'org.jboss.spec.javax.rmi:jboss-rmi-api_1.0_spec:1.0.6.Final'
+    compileOnly 'org.apache.logging.log4j:log4j-core:2.11.1'
+    runtimeOnly 'io.jsonwebtoken:jjwt-impl:0.10.5'
+    runtimeOnly('io.jsonwebtoken:jjwt-jackson:0.10.5') {
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
+    }
+    runtimeOnly 'com.sun.xml.messaging.saaj:saaj-impl:1.5.2'
+    runtimeOnly 'org.jvnet.staxex:stax-ex:1.8.3'
+    testImplementation 'commons-io:commons-io:2.6'
+    testCompileOnly 'org.hamcrest:hamcrest-all:1.3'
+    testImplementation 'junit:junit:4.12'
+    testImplementation 'org.apache.httpcomponents:fluent-hc:4.5.3'
+    testImplementation('org.elasticsearch.plugin:reindex-client:7.10.2') {
+        exclude group: 'org.elasticsearch', module: 'elasticsearch-ssl-config'
+        exclude group: 'org.apache.httpcomponents', module: 'httpclient'
+    }
+    testImplementation 'org.elasticsearch:elasticsearch-ssl-config:7.10.2'
+    testImplementation 'org.elasticsearch.plugin:percolator-client:7.10.2'
+    compile 'org.elasticsearch.plugin:lang-mustache-client:7.10.2'
+    compile 'org.elasticsearch.plugin:parent-join-client:7.10.2'
```

Comment:
Group all testImplementation, implementation, compileOnly, runtimeOnly dependencies

Reply:
By group, do you mean put dependencies with same scope together?

Reply:
Done